shell/platform/windows/flutter_project_bundle.cc

```diff
@@ -80,8 +80,17 @@ UniqueAotDataPtr FlutterProjectBundle::LoadAotData(
 
 FlutterProjectBundle::~FlutterProjectBundle() {}
 
+void FlutterProjectBundle::SetSwitches(
+    std::vector<std::string> const& switches) {
```

Comment:
Nit: we generally put `const` to the left of the type (so, `const std::vector<...>& foo`)

Applies throughout the PR.